src/main.rs

```diff
@@ -86,6 +86,8 @@ async fn main() {
 }
 
 async fn run() -> Result<()> {
+    solana_logger::setup_with_default("solana=info");
```

Comment:
Would it be better to leave it off by default? Just in case any dependency start logging messages at some point.

Reply:
Agreed.